eknigge/GuideWalker
# Sidebar

# Description of Change

This PR addresses the following issues:

Closes #24 - Sidebar/Drawer App Management UI

Adds a navigation/hamburger button on the top left. When the navigation button is pressed, a side drawer will become visible with additional screen options (account, create a tour, signout). 

<img width="324" alt="Screen Shot 2021-11-18 at 5 00 23 PM" src="https://user-images.githubusercontent.com/47171828/142671836-410ebdfe-4cf0-41e5-8cbc-3688b3a74f28.png">


<img width="321" alt="Screen Shot 2021-11-18 at 5 00 53 PM" src="https://user-images.githubusercontent.com/47171828/142671898-00f65c3e-9a5b-48e5-a345-0d02f40beeb3.png">


## How to test

1. Run the app
2. A navigation button will appear on the top left.
3. Press the navigation button, a side drawer should become visible. 
4. The side drawer should only partially cover the map, press anywhere on the map and the side drawer should retract. 
5. Select the different screens links in the navigation to be taken to a different screen, select the Guidewalker link in the navigation to be taken back to the map.

## Checklist

- [x] Did you request a review from someone?
